x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/analysis/PreAnalyzer.java

```diff
@@ -23,11 +23,11 @@ public static class PreAnalysis {
         public static final PreAnalysis EMPTY = new PreAnalysis(emptyList(), emptyList());
 
         public final List<TableInfo> indices;
-        public final List<String> policyNames;
+        public final List<Enrich> unresolvedEnriches;
```

Comment:
Everything is unresolved in the preAnalyzer - I suggest using `policies` or `enrichPolicies` instead or enriches.

Reply:
++, I have renamed this to `enriches` in https://github.com/elastic/elasticsearch/pull/104840/commits/11d4d7ef197546fd3693af0cbd5db7a3a5f4431c